proto/oak_functions/service/oak_functions.proto

```diff
@@ -57,10 +58,12 @@ message InitializeRequest {
 }
 
 message InitializeResponse {
-  PublicKeyInfo public_key_info = 1;
+  PublicKeyInfo public_key_info = 1 [deprecated = true];
+  oak.attestation.v1.Evidence evidence = 2;
 }
 
 message PublicKeyInfo {
+  option deprecated = true;
```

Comment:
> It also deprecates the PublicKeyInfo message, which is still needed untill we have a DICE verification library, but will be removed when clients no longer rely on direclty fetching the public key.

Do you mean when we don't have the evidence verification on the client? I think it's still a very long way to go. So we might deprecate the method later.

Also, this method will still be used by the untrusted app to send the evidence to the KPS. So not sure whether it should be deprecated.

Reply:
We should not send the raw public keys around. The evidence already contains the public key in a certificate, and any component that needs to use the public must extract it from the evidence for the key to be trusted. For now we should focus on a verification library that can just extract the public key, and over time do more and more verification over time. At least then the code will follow the right pattern even if it does not yet do full verification.